AUTHORS

```diff
@@ -1,3 +1,3 @@
 Nicholas H.Tollervey (ntoll@ntoll.org)
 Matt Wheeler (m@funkyhat.org)
-
+L0C4RD (L0C4RD@gmail.com)
```

Comment:
Given this is software written for a device aimed at kids, we need your real name. No need to be shy about your contribution either. ;-)